clang/test/Preprocessor/predefined-arch-macros.c

```diff
@@ -4141,6 +4141,20 @@
 
 // Begin SystemZ/GCC/Linux tests ----------------
 
+// RUN: %clang -E -dM %s -o - 2>&1 \
+// RUN:     -target s390x-ibm-zos \
```

Comment:
`--target=` for new driver tests.

However, this does not test clangDriver and therefore `%clang_cc1` is preferred.

(You can ignore previous lines that obey the convention. I can fix them later.)